index.html

```diff
@@ -231,11 +231,13 @@
 
 # Thank you
 
-Meili
+Github: mei-li
 
-meili.tr@gmail.com
+Email: meili.tr@gmail.com
 
-@meilitr
+Twitter: @meilitr
+
```

Comment:
Thank you for your time and the presentation about open source was very helpful to me.Claudine